common/common-rest/src/main/java/org/apache/servicecomb/common/rest/definition/RestOperationMeta.java

```diff
@@ -105,6 +110,16 @@ public void init(OperationMeta operationMeta) {
     setAbsolutePath(concatPath(swagger.getBasePath(), operationMeta.getOperationPath()));
   }
 
+  private void checkDownloadFile(Operation operation) {
+    try {
+      Response response = operation.getResponses().get("200");
+      downloadFile = response.getSchema().getType().toLowerCase().equals("file");
+    } catch (Exception e) {
```

Comment:
Do not use exception to do flow control. Check null is better.

Reply:
ok